spec/Functions.tex

```diff
@@ -1047,14 +1047,13 @@ \section{Where Expressions}
 \index{expressions!where@\chpl{where}}
 \index{functions!where@\chpl{where}}
 
-The instantiation of a generic function can be constrained by {\em
-where clauses}.  A where clause is specified in the definition of a
-function~(\rsec{Function_Definitions}).  When a function is
-instantiated, the expression in the where clause must be a parameter
-expression and must evaluate to either \chpl{true} or \chpl{false}.
-If it evaluates to \chpl{false}, the instantiation is rejected and the
-function is not a possible candidate for function resolution.
-Otherwise, the function is instantiated.
+The list of function candidates can be constrained by {\em where clauses}.  A
+where clause is specified in the definition of a
+function~(\rsec{Function_Definitions}).  When a function is resolved, the
+expression in the where clause must be a parameter expression and must
+evaluate to either \chpl{true} or \chpl{false}.  If it evaluates to
```

Comment:
Going with:

`The expression in the where clause must be a boolean parameter expression that evaluates to either "true" or "false"`